.github/workflows/build-test.yaml

```diff
@@ -30,7 +30,7 @@ jobs:
           key: ubuntu-x86-64-target-cache-nightly
       - name: Install dependencies
         run: |
-          cargo install cargo-hack
+          cargo install cargo-hack dprint
```

Comment:
Do we also check the formatting later?

Reply:
It should be checked when `cd polarsl make fmt` runs.

Reply:
Check. 👍

Reply:
I see now what you mean. Fixed here I hope: https://github.com/pola-rs/polars/pull/1733. I assumed it would also use the Makefiles.